stores/appStore.ts

```diff
@@ -135,7 +135,7 @@ export const useAppStore = defineStore("app", {
         } finally {
           this.downloading = false;
         }
-      })();
+      }, 200)(); // include small delay to avoid tooltip flicker on immediate button disable
```

Comment:
This change seemed to be necessitated by the css changes - not really sure how, but I think the issue was with immediately disabling the button while the tooltip was showing - the tooltip was momentarily displayed orphaned from the button on top left of page.